Add is_csv kwarg to prepare_upload util (#165)
alphagov/notifications-python-client: CHANGELOG.md

## 5.6.0

> * Add support for optional parameter `is_csv` in `prepare_upload()` file upload utility function. This fixes a bug to ensure that the file is downloaded as a CSV rather than a TXT file.

Reviewer: ```suggestion
* Fix a bug in Send a file by email. Add support for an optional `is_csv` parameter in the `prepare_upload()` utility. This ensures that the file is downloaded as a CSV rather than a TXT file.
```

Author: Looks good to me!

Reviewer 2: Thanks Karl. 

Wonder if it should be 'in the `prepare_upload()` function' or just 'in `prepare_upload()`' rather than 'in the `prepare_upload()` utility' though? I don't think the word utility is meaningful

Author: That does make better sense to me - I've updated with your initial suggestion there.  Does that sound OK?

Reviewer: That's where your subject matter expertise comes in very handy @idavidmcdonald.

'Utility' isn't meaningful to me either, but I wasn't sure if it was a specific term.

Let's go with your suggestion.

> * Fix a bug in Send a file by email. Add support for an optional `is_csv` parameter in the `prepare_upload()` function. This ensures that the file is downloaded as a CSV rather than a TXT file.

Reviewer 2: Final edit. Just spoke with Karl and we think the sentences are too disjoint so we now suggest:

Add support for an optional is_csv parameter in the prepare_upload() function. This fixes a bug when sending a CSV file by email. This ensures that the file is downloaded as a CSV rather than a TXT file.

After that then this is definitely going to be good to go. Thanks for your patience!
